### PR TITLE
Initial Commit Project added.

### PR DESCRIPTION
**Pull request by ipstushard**
I have created new branch Develop and need to merge in main.
I have created Basic project and bind Grid data from Devexpress.
Will do to make reusable that grid.

### CODE REVIEW

File: MVCDemoApp/Models/SampleData.cs [hunk 1]
```diff
@@ -0,0 +1,365 @@
+using System;
```
Comment (ShubhamG2908) [inline MVCDemoApp/Models/SampleData.cs:1]: Remove this file. Please check the ReadMe file and add Model accordingly.

File: NuGet.config [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ShubhamG2908) [inline NuGet.config:1]: Remove this file. We don't need this.

**Review by ShubhamG2908 — CHANGES_REQUESTED** [review]
Added Comment

File: MVCDemoApp/Models/SampleData.cs (hunk 1 above)
Comment (ipstushard) [reply MVCDemoApp/Models/SampleData.cs]: Managed it

File: NuGet.config (hunk 2 above)
Comment (ipstushard) [reply NuGet.config]: Removed it

**Pull request merged**